types/lodash/index.d.ts

```diff
@@ -280,9 +280,9 @@ declare namespace _ {
         (value: number): LoDashImplicitWrapper<number>;
         (value: string): LoDashImplicitStringWrapper;
         (value: boolean): LoDashImplicitWrapper<boolean>;
-        (value: number[]): LoDashImplicitNumberArrayWrapper;
-        <T>(value: T[]): LoDashImplicitArrayWrapper<T>;
-        <T extends {}>(value: T): LoDashImplicitObjectWrapper<T>;
+        (value: number[] | null | undefined): LoDashImplicitNumberArrayWrapper;
+        <T>(value: T[] | null | undefined): LoDashImplicitArrayWrapper<T>;
+        <T extends {}>(value: T | null | undefined): LoDashImplicitObjectWrapper<T>;
```

Comment:
This lines aren't quite true. You're saying that when I pass `undefined` or `null` that I get one of these wrappers but that is not true. `undefined` will never give me something that is usable as an `LoDashImplicitArrayWrapper<T>`.

Also all three match `undefined` so which one should be used?